packages/client/bin/cli.ts

```diff
@@ -344,6 +344,16 @@ const args: ClientOpts = yargs(hideBin(process.argv))
       'To run client in single node configuration without need to discover the sync height from peer. Particularly useful in test configurations. This flag is automically activated in the "dev" mode',
     boolean: true,
   })
+  .option('vmProfileBlocks', {
+    describe: 'Report the VM profile after running each block',
+    boolean: true,
+    default: false,
+  })
+  .option('vmProfileTxs', {
+    describe: 'Report the VM profile after running each block',
+    boolean: true,
+    default: false,
+  })
```

Comment:
Oh this is cool! 🤩

Reply:
Didn't know that you had integrated so deeply into client already...

Reply:
Yes, the only thing not to forget is; it does not work if you don't set the `DEBUG=ethjs,evm:profiler` env, so this is something we can improve (but not sure how to do this cleanly)

Reply:
You should just be able to set the debug variable programmatically with this `process.env.DEBUG = 'ethjs,evm:profiler'`.  So just add a code block to the `run` command in the client that checks for either of these args and then sets the debug accordingly.  Or better, do something like this:
```js
if (args.vmProfileBlocks || args.vmProfileTxs) {
  let debugString = process.env.DEBUG
  if debugString.startsWith('ethjs') {
    debugString = debugString + ',evm:profiler'
  } else {
    debugString = 'ethjs,evm:profiler'
  }
  process.env.DEBUG=debugString
}   
```

Reply:
Note, I haven't tested the above code but I do something like this in one of the `trie` demos to force it to output certain debug logs.